spec/index.html

```diff
@@ -17,7 +18,7 @@
     <script src="/testem.js"></script>
 
     <!-- include source files here... -->
-    <script src="/lib/annotator-custom-editor.js"></script>
+    <script src="/.tmp/annotator-custom-editor.js"></script>
```

Comment:
`.tmp` serves the files when you're developing and running the tests. you run a final deploy with inline source-maps to the `lib` folder.